app/v2/controllers/V2MovementsController.scala

```diff
@@ -124,7 +130,23 @@ class V2MovementsControllerImpl @Inject() (
           movementResponse <- persistAndSendToEIS(request.body, MovementType.Departure, MessageType.DeclarationData)
         } yield movementResponse).fold[Result](
           presentationError => Status(presentationError.code.statusCode)(Json.toJson(presentationError)),
-          response => Accepted(HateoasNewMovementResponse(response.movementId, response.boxResponse, MovementType.Departure))
+          response => Accepted(HateoasNewMovementResponse(response.movementId, response.boxResponse, None, MovementType.Departure))
+        )
+    }
+
+  private def submitDepartureDeclarationLargeXML(): Action[Source[ByteString, _]] =
+    (authActionNewEnrolmentOnly andThen messageSizeAction()).async(streamFromMemory) {
```

Comment:
@drnaylor-oc do we need to include andThen messageSizeAction()) ? Because we are not passing any payload here so we should not include it? if really want to include it then basically we need to customize that action and if payload is not empty in case of large message we should report error to end user?

Reply:
Yeah, that's a good point, `messageSizeAction` shouldn't be here at all.

Reply:
Don't think the code reaches here - If I removed contentLength header - I always get `501 Not Implemented` - anyways, will remove the action `messageSizeAction`